src/Compilers/Core/Portable/Operations/OperationKind.cs

```diff
@@ -142,6 +142,10 @@ public enum OperationKind
         DynamicObjectCreationExpression = 0x125,
         /// <summary>Indicates an <see cref="IDynamicMemberReferenceExpression"/>.</summary>
         DynamicMemberReferenceExpression = 0x126,
+        /// <summary>Indicates an <see cref="IQueryExpression"/>.</summary>
+        QueryExpression = 0x127,
+        /// <summary>Indicates an <see cref="IOrderingExpression"/> in an <see cref="IOrderByQueryClause"/>.</summary>
+        OrderingExpression = 0x128,
```

Comment:
>OrderingExpression [](start = 8, length = 18)

I am not sure why Ordering are singled out as an expression. What makes it special by comparison to other clauses? Can ordering be stand-alone, without query expression?